timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -175,19 +174,19 @@ def run(self):
         return output_string
 
 
-class LinuxRulesSigmaPlugin(SigmaPlugin):
+class SampleRulesSigmaPlugin(SigmaPlugin):
```

Comment:
Do we need this? Is it for testing? Sample of what? :)
It clutters the UI with this in the available analyzer list.

Reply:
agree

Reply:
Yeah so the rationale behind is that we have no specific folder for OS, so I assumed it is better to have one for developing rules that would not be pushed to github.

However I see the issue here.
Would it make more sense if we say we have one Analyzer in the UI and it grabs all rules in the folder?

Reply:
fixed